src/index.html

```diff
@@ -13,5 +13,166 @@
   <body>
     <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post"
+    onsubmit="onSubmit()">
+
+      <!-- personal -->
+      <fieldset class="fieldset">
+        <legend> Personal information</legend>
+        <label class="form-field" for="surname">
+          Surname:
+          <input type="text"
+          autocomplete="false"
+          name="surname"
+          id="surname"
+          minlength="3"
+          maxlength="10">
+        </label>
+
+        <label class="form-field" for="name">
+          Name:
+          <input type="text"
+          autocomplete="false"
+          name="name"
+          id="name">
+        </label>
+
+        <label class="form-field" for="age">
+          How old are you?
+          <input type="number"
+          value="12"
+          name="age"
+          max="100"
+          min="12"
+          id="age">
+        </label>
+
+        <label class="form-field" for="birth">
+          Full date of birth:
+          <input type="date"
+          name="birthday"
+          id="birth">
```

Comment:
```suggestion
          name="birthday"
          id="birthday">
```
You can use the same names if you want